CefSharp/ModelBinding/ModelBindingExtensions.cs

```diff
@@ -67,7 +66,17 @@ public static bool IsArray(this Type source)
         /// <returns>return if collection, array or enumerable</returns>
         public static bool IsCollectionOrArray(this Type source)
         {
-            return source.IsCollection() || source.IsArray() || source.IsEnumerable();
+            return !source.IsGenericObject() && (source.IsCollection() || source.IsArray() || source.IsEnumerable());
+        }
+
+        /// <summary>
+        /// The type is a generic javascript object (dictionary)
+        /// </summary>
+        /// <param name="source">source type</param>
+        /// <returns>return true if is a generic js object</returns>
+        public static bool IsGenericObject(this Type source) 
```

Comment:
The name of this method is confusing, I look at it and think of `.Net generics`, I don't actually think it's required either if we make a few other simplifications.